langchain4j/src/test/java/dev/langchain4j/service/ServiceOutputParserTest.java

```diff
@@ -203,4 +203,27 @@ void outputFormatInstructions_PersonWithFinalFields() {
                         "\"birthDate\": (type: date string (2023-12-31)),\n" +
                         "}");
     }
+
+    static class PersonWithParents {
+        private String firstName;
+        private String lastName;
+        private List<PersonWithParents> parents;
```

Comment:
Could you please add separate tests for `Parent[]` and `Parent mother; Parent father` cases?
Thanks!

Reply:
outputFormatInstructions_PersonWithMotherAndFather have been added

Reply:
Thanks! Could you please also add one test for `Parent[]` case since it has a different handling logic?